yb-voyager/src/callhome/diagnostics.go

```diff
@@ -53,7 +53,7 @@ CREATE TABLE diagnostics (
 	migration_phase TEXT,
 	phase_payload JSONB,
 	migration_type TEXT,
-	time_taken_sec NUMERIC(20,2),
+	time_taken_sec bigint,
```

Comment:
even `int`or `unsigned int` is enough here.
Because `2147483647 seconds` --> `~68 years`

Reply:
yeah, makes sense. Made the change